### DISCUSSION TITLE
How to load ssl-log-ext.zeek?

### DISCUSSION THREAD

**Discussion by yksee** [Security-Onion-Solutions/securityonion#8036] [Unsupported Versions] [answered]
Zeek has a script called policy/protocols/ssl/ssl-log-ext.zeek, which adds additional information to the SSL log.  This is not loaded by default. (Refer to https://docs.zeek.org/en/master/scripts/policy/protocols/ssl/ssl-log-ext.zeek.html)

Under Security Onion, there is a way to load custom scripts (Refer to https://docs.securityonion.net/en/2.3/zeek.html) by editing /opt/so/saltstack/local/pillar/global.sls. I have added "- protocols/ssl/ssl-log-ext" to the zeek->local->'@load' section and restart Zeek:

Can I check if the above is the correct way to enable ssl-log-ext in SO?

**Comment by dougburks** [comment, marked as answer]
Seems like you're on the right track. If you want to verify that the script got loaded, then you could restart Zeek and check `/nsm/zeek/logs/current/loaded_scripts.log` as discussed at https://docs.securityonion.net/en/2.3/zeek.html:
> One the configuration has been updated, Zeek can be restarted with sudo so-zeek-restart on applicable nodes to pick up the changes. Finally, /nsm/zeek/logs/current/loaded_scripts.log can be checked to ensure the new module has been loaded.
